programs/bench.c

```diff
@@ -709,44 +720,60 @@ BMK_return_t BMK_benchMemAdvanced(const void* srcBuffer, size_t srcSize,
     const void ** const srcPtrs = (const void** const)malloc(maxNbBlocks * sizeof(void*));
```

Comment:
nit : 
the final `const` in variable declaration is useful : it states the variable cannot change after its creation, which is great for reviewer, since it's not longer necessary to track these variables in case they would change later in the function.

But in the casting, `(type const)`, such final `const` does not achieve anything useful.